Source/Foundation/bsfUtility/Private/UnitTests/BsUtilityTest.cpp

```diff
@@ -3,14 +3,48 @@
 #include "Testing/BsConsoleTestOutput.h"
 #include "Private/UnitTests/BsUtilityTestSuite.h"
 
+#include "BsApplication.h"
```

Comment:
Generally I organize the non-plugin code in layers. 'Utility' layer is the lowest one, and it shouldn't be referencing anything from the higher layers. Are these tests failing without the Application started up? Ideally we can run them without it.

Reply:
hm. well we could just change it to BsCoreApplication, which is the core layer, and therefore compatible with all other layers of testing for most part. I'll make the change and see if that works.

Reply:
nope nvm, CoreApplication can't be initialized by itself for running the utility tests. stuff like gProfiler still gets called